src/common/options.cc

```diff
@@ -3673,7 +3673,7 @@ std::vector<Option> get_global_options() {
     // filestore
 
     Option("filestore_rocksdb_options", Option::TYPE_STR, Option::LEVEL_DEV)
-    .set_default("compaction_readahead_size=2097152")
+    .set_default("max_background_compactions=8;compaction_readahead_size=2097152")
```

Comment:
nit, can we use `,` as the separator? to be consistent with other `_rocksdb_options`.

Reply:
even though, "max_background_compactions" is still supported, but since rocksdb 5.6.0, `max_background_compactions` is replaced with `max_background_jobs`. to have 8 background compactions, we can specify "max_background_jobs=10", and 2 jobs will be allocated for "max_background_flushes".

Reply:
okay, it's more backport-friendly. we can update it in master after it gets merged.